src/pages/Search/MadLibsSearch.vue

```diff
@@ -264,10 +310,47 @@ export default {
 
 .search-statement-wrapper {
 	margin-bottom: rem-calc(32);
+	padding: 0.375rem;
+}
+
+.madlibs-dropdown {
+	background-image: url('../../assets/images/medium-chevron2x.png');
```

Comment:
please update to you relative syntax:
`ex. background-image: url('~@/assets/images/whatever-file-name.ext');`